### DISCUSSION TITLE
How to re-enable github pages after unpublishing?

### DISCUSSION THREAD

**Discussion by rvzaytsev** [community/community#139396] [Actions]
### Select Topic Area

Question

### Body

According to [documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/unpublishing-a-github-pages-site#re-enabling-a-site-that-has-been-unpublished) these are the instructions:

> A successful workflow run in the repository for your site will create a new deployment. Trigger a workflow run to redeploy your site.

How do I trigger the workflow run? Can you update the documentation with proper detailed instructions instead of this GFY comment?

**Comment by davevad93** [comment]
Hi @rvzaytsev ,

You can re-trigger the workflow run of your GitHub Pages website by going to the `Actions` tab of your repo, click on the `pages build and deployment` workflow, then click on `Re-run all jobs` located in the upper right corner of the page.

**Reply by rvzaytsev** [reply]
Where exactly is that button? Here is a screenshot of what I see:
![Screenshot from 2024-09-22 10-15-15](https://github.com/user-attachments/assets/c75672a2-2d37-4206-963e-b2a83e5192fd)

**Reply by davevad93** [reply]
Hey @rvzaytsev , It's located in the upper right corner of the page, as in the screenshot below:
<img width="990" alt="5555" src="https://github.com/user-attachments/assets/73c36812-5b1a-4c36-807b-3bce601c1e2d">

Make sure that the deployment is enabled in the repo settings, otherwise you won't be able to re-run the job.

<img width="594" alt="00000" src="https://github.com/user-attachments/assets/cb75802d-1edf-48fb-9e04-2735783b4d36">

**Comment by rvzaytsev** [comment]
Ohh, I see, after changing the settings the button appeared. Thank you so much!!

**Reply by davevad93** [reply]
You're welcome @rvzaytsev 👍
